root/static/scripts/relationship-editor/common/entity.js

```diff
@@ -6,6 +6,18 @@
 var mergeDates = require('./mergeDates.js');
 var deferFocus = require('../../edit/utility/deferFocus.js');
 
+function getDirection(relationship, source) {
```

Comment:
Direction is relative to the entity you're editing from, which is what `source` is.